docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+* The ``sktime`` project
+* other sources and influences mentioned above
+
+
+Appendix A: Guidance in applying the CoC
+----------------------------------------
+
+This appendix provides some additional guidance in applying the CoC.
+It contains sections on:
+
+* how to apply the CoC in some common situations where principles might conflict
+* investigating CoC violations in resource allocation
+
+A.1 Resolving conflicting CoC principles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes, CoC requirements end up in a trade-off, or in conflict with each other.
+We outline a few guiding examples in how these should be traded off.
+
+Examples:
+
+* contributor criticizing leadership for handing resources to personal network, leadership invoking "discomfort"
+* passionate disagreement on technical content between contributors
+* accidental violations of CoC causally due to protected characteristics of the person violating the CoC
+* systemic discrimination issues whose resolution would cause discomfort in the community
+
+Guidance rules:
+
+* the less powerful party should be protected, i.e., CoC cannot be invoked by lead community member to silence critique of how funds are spent.
+  But, CoC should protect new contributors or early career members from more powerful community members, e.g., in operational or technical disputes.
+* technical content is not invalidated by tone, but tone must remain civil at all times.
+  I.e., valid arguments are not discarded because of tone, but participants in a technical
+  dispute may be punished for tone that is unwelcoming, e.g., name calling etc.
+* conditions that make compliance with certain aspects of CoC more difficult should be taken into account,
+  especially in parties that are the less powerful in a dynamic. Common examples are certain forms of neuroatypicality.
+  A more illustrative but very rare example would be Tourette's with the rare swearing tic that cannot be "turned off".
+* issues of systemic exclusion or discrimination should be addressed when apparent, even at the cost of some community discomfort.
+  E.g., meetings should be moved to less convenient times, or processes made more "formal",
+  if not doing so would exclude others in a discriminatory fashion.
+
+Behaviour that is clearly racist, sexist, etc, is always a CoC violation, and never part of trade-offs.
+
+
+A.2 Guidance on applying the CoC for resource allocation decisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section deals with resource allocation decisions by ``sktime`` community members,
+and criteria for decision making on resource allocation violating the code of conduct.
+
+Scope
+^^^^^
+
+In scope for this section are all resource allocation decisions that satisfy at least one of the following criteria:
+
+* any decisions that allocate resources primarily obtained due to the ``sktime`` project and the volunteer contributions of the community.
```

Comment:
I see what you mean and how it could be construed to be in conflict with the BSD-3 license.

It is meant to apply to elected members only, i.e., core developers. It is not 100% clear from the text, so I will clarify that.

I.e., the gist should be, you are elected to represent `sktime` (as core dev, CC member etc), so you need to act on behalf of the community rather than on your own behalf.

If you are not an elected member, do as you wish as long as you comply with the BSD-3 license.

Reply:
I have clarified the scope of jurisdiction.

Reply:
see above - I think it should now be 100% clear that the software distribution license is not impacted, that this applies primarily to elected members of communities in executing their roles (or operating in grey spaces between their sktime role and other roles)

Reply:
I don't think core-devs bear such burden, do they have access to sktime funds / OC? I thought that's the responsibility that only CC has? Acting on behalf of the community, etc.

Reply:
" obtained due to the ``sktime`` project" is hard to define. For example, if a person gets hired on a position and one of the points of their resume are "a core-dev of sktime" - do they now obtain their salary due to sktime project and have to give up some of it? Maybe a start-up project is named "sktime product demand forecasting" and gets investment, how to define if the investors made their decision primarily due to "sktime" in the name and not the "product demand forecasting" part?

Reply:
> I don't think core-devs bear such burden, do they have access to sktime funds / OC? I thought that's the responsibility that only CC has? Acting on behalf of the community, etc.

Core devs have a lesser version of rights of this kind, but they still do have them, e.g., when organizing an event, handling funds, handling social media, etc, on behalf of the community or in delegation.

Formally, if you look at our main gov document, *core developers participate in decision making*.

So, of course, rules on decision making should include them in theie key scopes, in my opinion.